File: 231109/숫자의 계절은/season-of-num.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int a;
    cin >> a;
    if(a >= 12 || a <= 2) {
        cout << "Winter";
    }
    else if(a <= 5) {
        cout << "Spring";
    }
    else if(a <= 8) {
        cout << "Summer";
    }
    else {
        cout << "Fall";
    }
    return 0;
}

File: 250211/수의 계절은/season-of-num.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();

        if(a >= 12 || a <= 2) {
            System.out.print("Winter");
        }
        else if(a <= 5) {
            System.out.print("Spring");
        }
        else if(a <= 8) {
            System.out.print("Summer");
        }
        else if(a <= 11) {
            System.out.print("Fall");
        }
    }
}